pipeline/metadata/6.0.yaml

```diff
@@ -7,6 +7,23 @@
 # parameter required to create a similar custom file is suite name, suite yaml file, global configuration file,
 # platform, rhbuild, inventory and metadata information like frequency of execution, tier, cloud type, functional group and stage.
 #=====================================================================================
+- overrides:
```

Comment:
IMO, we could do
```
overrides:
  ibmc:
    custom-config:
      - "grafana_image=ceph-qe-registry.syd.qe.rhceph.local/rh-osbs/grafana:ceph-6-rhel-9-containers"
      - "promtail_image=ceph-qe-registry.syd.qe.rhceph.local/rh-osbs/promtail:ceph-6-rhel-9-containers"
   openstack:
     custom-config:
       - "grafana_image=regstry-proxy.engineering.redhat.com/rh-osbs/grafana:ceph-6-rhel-9-containers"
       - "promtail_image=regstry-proxy.engineering.redhat.com/rh-osbs/promtail:ceph-6-rhel-9-containers"
```
Other keys can also be supported (and probably a bit readable).

Reply:
Done. other keys could be supported with my previous logic as well. But I agree this one is more readable. I have made the changes, pls let me know if this looks good.